test/functional/test_runner.py

```diff
@@ -89,212 +89,212 @@
-    "feature_consortium.py",
-    "interface_http.py",
-    "interface_http_cors.py",
-    "interface_rpc.py",
-    "rpc_psbt.py",
-    "rpc_users.py",
-    "feature_proxy.py",
-    "rpc_signrawtransaction.py",
-    "wallet_groups.py",
-    "p2p_disconnect_ban.py",
-    "rpc_decodescript.py",
-    "rpc_blockchain.py",
-    "rpc_deprecated.py",
-    "wallet_disable.py",
-    "rpc_net.py",
-    "wallet_keypool.py",
-    "p2p_mempool.py",
-    "rpc_setban.py",
-    "p2p_blocksonly.py",
-    "mining_prioritisetransaction.py",
-    "p2p_invalid_locator.py",
-    "p2p_invalid_block.py",
-    "p2p_invalid_messages.py",
-    "p2p_invalid_tx.py",
-    "feature_foundation_migration.py",
-    "feature_assumevalid.py",
-    "example_test.py",
-    "feature_jellyfish_initial_funds.py",
-    "wallet_txn_doublespend.py",
-    "wallet_txn_clone.py --mineblock",
-    "feature_notifications.py",
-    "rpc_getblockfilter.py",
-    "rpc_invalidateblock.py",
-    "feature_rbf.py",
-    "mempool_packages.py",
-    "mempool_package_onemore.py",
-    "rpc_createmultisig.py",
-    "feature_versionbits_warning.py",
-    "rpc_preciousblock.py",
-    "wallet_importprunedfunds.py",
-    "p2p_leak_tx.py",
-    "rpc_signmessage.py",
-    "wallet_balance.py",
-    "feature_nulldummy.py",
-    "wallet_import_rescan.py",  # nodes = 6
-    "wallet_import_with_label.py",
-    "rpc_bind.py --ipv4",
-    "rpc_bind.py --ipv6",
-    "rpc_bind.py --nonloopback",
-    "mining_basic.py",
-    "wallet_bumpfee.py",
-    "wallet_bumpfee_totalfee_deprecation.py",
-    "rpc_named_arguments.py",
-    "wallet_listsinceblock.py",
-    "p2p_leak.py",
-    "feature_higher_collateral_factor.py",
-    "feature_skip_collateral_factor_check.py",
-    "wallet_encryption.py",
-    "feature_dersig.py",
-    "feature_cltv.py",
-    "rpc_updatemasternode.py",
-    "rpc_uptime.py",
-    "feature_longterm_lockin.py",
-    "wallet_resendwallettransactions.py",
-    "feature_custom_poolreward.py",
-    "wallet_fallbackfee.py",
-    "feature_token_lock.py",
-    "feature_minchainwork.py",
-    "rpc_getblockstats.py",
-    "feature_median_time.py",
-    "wallet_create_tx.py",
-    "p2p_fingerprint.py",
-    "feature_uacomment.py",
-    "wallet_coinbase_category.py",
-    "feature_filelock.py",
-    "p2p_unrequested_blocks.py",
-    "rpc_listaccounthistory.py",
-    "feature_listaccounthistory_multiaccountquery.py",
-    "rpc_getaccounthistory.py",
-    "feature_includeconf.py",
-    "rpc_deriveaddresses.py",
-    "rpc_deriveaddresses.py --usecli",
-    "rpc_scantxoutset.py",
-    "rpc_getcustomtx.py",
-    "rpc_listvaulthistory.py",
-    "feature_logging.py",
-    "feature_loan_scheme.py",
-    # 'feature_forced_reward_address.py',
-    "feature_loan_vault.py",
-    "feature_loan_deposittovault.py",
-    "feature_loan_interest.py",
-    "feature_loan_estimateloan.py",
-    "feature_loan_priceupdate.py",
-    "feature_loan_vaultstate.py",
-    "feature_loan.py",
+    # "mining_getblocktemplate_longpoll.py",
```

Comment:
I do not think you wanted to do this. What I do is have a script file with the following...

```
./test/functional/test_runner.py \
    feature_address_map.py \
    feature_evm.py \
    feature_evm_contract_env_vars.py \
    feature_evm_contracts.py \
    feature_evm_dfi_intrinsics.py \
    feature_evm_dst20.py \
    feature_evm_fee.py \
    feature_evm_logs.py \
    feature_evm_miner.py \
    feature_evm_proxy.py \
    feature_evm_rollback.py \
    feature_evm_rpc.py \
    feature_evm_rpc_fee_history.py \
    feature_evm_rpc_filters.py \
    feature_evm_rpc_transaction.py \
    feature_evm_smart_contract.py \
    feature_evm_transaction_replacement.py \
    feature_evm_transferdomain.py \
    feature_evm_vmmap_rpc.py
```

Reply:
oops... 😪